### PR TITLE
Upgrade ttl support for schedule and workflow

### LINKED ISSUE #2922: Dashboard: support TTL for archived schedule and workflow data
## Feature Request

**Is your feature request related to a problem? Please describe:**

At present, Dashboard supports setting TTL for archived events and experiments, but schedule and workflow do not.  We can extend to this [function](https://github.com/chaos-mesh/chaos-mesh/blob/master/pkg/dashboard/ttlcontroller/ttlcontroller.go#L73-L80)  to implement TTL for schedule and workflow data

**Describe the feature you'd like:**

**Describe alternatives you've considered:**

**Teachability, Documentation, Adoption, Migration Strategy:**

### PR DESCRIPTION
**Pull request by rksharma95**
### What problem does this PR solve?
1.  added `schedule` and `workflow` to the `ttlcontroller`.
2.  implements `DeleteByFinishTime` method for `workflow` store.

Close #2922

### Related changes

- [x] Need to update `chaos-mesh/website`
- [ ] Need to update `Dashboard UI`
- Need to **cheery-pick to release branches**
  - [ ] release-2.1
  - [ ] release-2.0

### Checklist

Tests

- [ ] Unit test
- [ ] E2E test
- [ ] No code
- [x] Manual test (add steps below)

Side effects

- [ ] `experimentstore` and `schedulestore` need to be refactored

### Release note 

```text
Please add a release note.

You can safely ignore this section if you don't think this PR needs a release note.
```

### DCO

If you find the DCO check fails, please run commands like below (Depends on the actual situations. For example, if the failed commit isn't the most recent) to fix it:

```shell
git commit --amend --signoff
git push --force
```

### CODE REVIEW

**Comment by STRRL** [comment]
Hi @rksharma95 , please execute `make check` then commit the changes to fix the `verify` CI check.

File: pkg/dashboard/core/workflow.go [hunk 1]
```diff
@@ -51,15 +51,16 @@ const (
...
-	Status    WorkflowStatus `json:"status,omitempty"`
-	Archived  bool           `json:"-"`
+	ID         uint           `gorm:"primary_key" json:"id"`
+	UID        string         `gorm:"index:workflow_uid" json:"uid"`
+	Namespace  string         `json:"namespace"`
+	Name       string         `json:"name"`
+	Entry      string         `json:"entry"` // the entry node name
+	CreatedAt  time.Time      `json:"created_at"`
+	EndTime    string         `json:"end_time"`
+	FinishTime time.Time      `json:"finish_time"`
```
Comment (cwen0) [inline pkg/dashboard/core/workflow.go:61]: You can use `EndTime` instead of `FinishTime`. In addition, I don't find where the logic to set `FinishTime`.

Comment (cwen0) [reply pkg/dashboard/core/workflow.go:61]: @STRRL PTAL

Comment (rksharma95) [reply pkg/dashboard/core/workflow.go:61]: @cwen0 thanks for your feedback. I will go with `EndTime` and to set it, I need to add logic to ` MarkAsArchived` and `MarkAsArchivedWithUID` methods in the `workflow` store, right?

Comment (STRRL) [reply pkg/dashboard/core/workflow.go:61]: Please use `EndTime` instead of `FinishTime` within workflow.

Comment (cwen0) [reply pkg/dashboard/core/workflow.go:61]: @rksharma95 You can just remove `FinishTime` and use `EndTime` directly.

**Review by cwen0 — APPROVED** [review]
LGTM

**Comment by cwen0** [comment]
@zhouqiang-cl @fewdan PTAL

**Review by fewdan — APPROVED** [review]
LGTM

**Pull request merged**